nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json

```diff
@@ -23,9 +17,11 @@
                         "type": "keyword"
                     },
                     "end_date": {
+                        "format": "strict_date_hour_minute_second",
                         "type": "date"
                     },
                     "start_date": {
+                        "format": "strict_date_hour_minute_second",
                         "type": "date"
```

Comment:
I'm wondering if/why we need funding dates with the second precision, if there are no particular reasons I'd convert these dates to `YYTT-MM-DD`.
I'd also propose to do this for for all of our dates where applicable.

Reply:
Yeah, that's fair. I'll convert to `strict_date` which is `YYYY-MM-DD`